src/css/editable.less

```diff
@@ -5,7 +5,6 @@
     width: 100%;
     padding: 2px;
     .box-sizing(border-box); // supported by IE8+: http://caniuse.com/css3-boxsizing
-    white-space: pre;
```

Comment:
thanks for pointing this is out---it's exactly why my first attempt a test case for this PR didn't work